torch/utils/checkpoint.py

```diff
@@ -458,14 +461,16 @@ def checkpoint(
         Output of running :attr:`function` on :attr:`*args`
     """
     if use_reentrant is None:
-        raise ValueError(
-            "torch.utils.checkpoint: please pass in use_reentrant=True or "
-            "use_reentrant=False explicitly. The default value of use_reentrant "
-            "will be updated to be False in the future. To maintain current "
-            "behavior, pass use_reentrant=True. It is recommended that you use "
-            "use_reentrant=False. Refer to docs for more details on the "
-            "differences between the two variants."
+        warnings.warn(
+            "torch.utils.checkpoint: the use_reentrant parameter should be "
+            "passed explicitly. In a future version of PyTorch we will error "
```

Comment:
if it's a "version" and not some future version, perhaps say explicitly which version?